pkg/features/kube_features.go

```diff
@@ -89,6 +89,14 @@ const (
 	// to take advantage of NoExecute Taints and Tolerations.
 	TaintBasedEvictions utilfeature.Feature = "TaintBasedEvictions"
 
+	// owner: @jcbsmpsn
+	// alpha: v1.7
+	//
+	// Gets a server certificate for the kubelet from the Certificate Signing
+	// Request API instead of generating one self signed and auto rotates the
+	// certificate as expiration approaches.
+	RotateKubeletServerCertificate utilfeature.Feature = "RotateKubeletServerCertificate"
```

Comment:
It's worth thinking about how this will look once this feature reaches GA. I would still expect to be able to provide a kubelet with a serving cert/key and have it just use it without trying to rotate. At that point, would we want a GA feature gate that someone could disable? I'm a little unclear about the longevity of feature gates once they reach GA.